art-community-platform/backend/app/api/helpers/recommendation_helpers.py

```diff
@@ -40,5 +40,66 @@ def get_tags_of_favourites(user):
     return tags
 
 
+#get 8 most popular users in the system
+def get_popular_users():
+    try:
+        user_list = User.objects.all()
+
+    except:
+        return [False, "couldnt get the users from database"]
+
+    #This is the list that is sorted by number of followers in decreasing order
+    new_list = sorted(user_list,key=lambda user: len(user.followers),reverse=True)
+
+
+    if len(new_list)<8:
+        return new_list
+    else:
+        new_list[:8]
+
+    return new_list
```

Comment:
There is a problem here (line 60), you shouldn't directly return the user list fetched from the database.
Instead, you should return users fetched using "user fetching helpers", because the database stores models in a special format and returning them directly is hard to handle in clients.
Also our clients (mobile and front end aren't ready for this)